src/cnxml/se.tsx

```diff
@@ -64,11 +64,15 @@ export default function serialize(
         ? content.pop()!
         : null
 
+    const documentAttrs = doc.className ? { class: doc.className } : {}
```

Comment:
You can just add `class={doc.className}` to JSX: this upgrade also includes a new XML serializer which filters out attributes whose `value == null`, so this trick is no longer needed.